apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -23,7 +23,7 @@ export class TaskCardUc {
 		let course: Course | undefined;
 
 		if (!this.authorizationService.hasAllPermissions(user, [Permission.TASK_CARD_EDIT])) {
-			throw new UnauthorizedException();
+			throw new ForbiddenException();
```

Comment:
so, when the user is not logged in, we get 401 - unauthorized response
this happens already from the controller, by using the [CurrentUser decorator](https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/authentication/decorator/auth.decorator.ts#L46)

so here in the UC, we check that the logged in user has permission. If not, we throw 403

to me, that seemed correct